src/components/treeEditor/views/BranchView.ts

```diff
@@ -0,0 +1,31 @@
+import NodeViewFactory from '../services/NodeViewFactory';
+import template from '../templates/branch.hbs';
+import NodeViewConfig from '../services/NodeViewConfig';
+
+export default Marionette.CollectionView.extend(
+    Object.assign(
+        {
+            initialize(options: { model: any, unNamedType?: string, stopNestingType?: string }) {
+                this.collection = options.model.get(options.model.childrenAttribute);
+            },
+
+            className: 'branch-item',
+
+            childView(childModel) {
+                return NodeViewFactory.getNodeView({
+                    model: childModel,
+                    unNamedType: this.options.unNamedType,
+                    stopNestingType: this.options.stopNestingType,
+                    forceBranchType: this.options.forceBranchType
+                });
+            },
+
+            childViewOptions() {
+                return _.omit(this.options, 'parent', 'model');
```

Comment:
you dont need omit
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_omit